<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkinBaker</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Bake/Unbake Skinning Information</string>
   </property>
   <widget class="QPushButton" name="ubButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Transfer baked information back to the skinCluster</string>
    </property>
    <property name="text">
     <string>Unbake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Bake skinning information on selected meshes</string>
    </property>
    <property name="text">
     <string>Bake</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ubDeleteSkinclusterCheckbox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Delete current skinCluster when unbacking</string>
    </property>
    <property name="text">
     <string>Delete skincluster</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ubIgnoreSceneNamespacesCheckbox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Will ignore namespaces from the scene when unbaking skinning information</string>
    </property>
    <property name="text">
     <string>Ignore scene namespace</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bSelectVertsButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select vertices that contain baked skinning information</string>
    </property>
    <property name="text">
     <string>Select vertices</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ubSmoothWeightsCheckbox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;(SLOW) Smooth vertices with no influences across the mesh topology.&lt;/p&gt;&lt;p&gt;To be used when the mesh topology changed (eg. Mesh got smoothed)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Smooth Weights</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bDeleteButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Bake skinning information on selected meshes</string>
    </property>
    <property name="text">
     <string>Clear baking</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>360</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Save to File</string>
   </property>
   <widget class="QPushButton" name="sfSaveButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sfDirectoryEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sfDirectoryButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sfRefreshButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Get current scene path</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sfUseObjectsNamesCheckbox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>65</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Use selected object(s) name as export filename</string>
    </property>
    <property name="text">
     <string>Filename equals object name</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Load from File(s)</string>
   </property>
   <widget class="QPushButton" name="lfLoadButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load single or multiple .weight files on the selected geometry</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lfIgnoreSceneNamespaceCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ignore scene namespace</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="lfDeleteSkinClusterCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete scene skinCluster</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="lfUseWorldspacePosCheckbox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Worldspace Positions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lfRemapperButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>65</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load single or multiple .weight files on the selected geometry</string>
    </property>
    <property name="text">
     <string>Load-&gt;Remap</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>445</y>
     <width>366</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string>Tools</string>
   </property>
   <widget class="QPushButton" name="tDeleteHistoryButton">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>20</y>
      <width>116</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete History</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>385</y>
     <width>366</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string>New SkinCluster Options</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>96</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalize Weights</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbNormalizeWeights">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>25</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Interactive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Post</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
